src/consent.ts

```diff
@@ -63,4 +63,35 @@ export class Consent {
     );
     document.head.prepend(script);
   }
+
+  async enableGoogleTagManager(
+    id: string,
+    role: ConsentCategory = "statistic"
+  ): Promise<void> {
+    await this.consentFor(role);
+    window.gtag({
+      "gtm.start": new Date(),
+      event: "gtm.js",
+    });
+    window.gtag("config", id, { send_page_view: false }); // page view is disabled because it's tracked via the analytics stimulus controller already
+    const script = document.createElement("script");
+    script.async = true;
+    script.setAttribute(
+      "src",
+      `https://www.googletagmanager.com/gtm.js?id=${id}`
+    );
+    document.head.prepend(script);
+
+    const noscript = document.createElement("noscript");
```

Comment:
Does it even make sense to add a `<noscript>` tag via JavaScript? What is the expected behaviour here?

This is running inside a script, so we definitely know that the user has enabled scripts, right?

Reply:
I spoke with jens yesterday and he said that we should have the no script regardless even though it's not used.

Reply:
What's the reasoning behind that?

Reply:
@JensRavens 

I believe that maybe it would hurt some sort of score by not having it on the website even though it's not used. 🤔

Reply:
it's for making sure that humans who "review" our rendered html don't complain about it missing. It has absolutely 0 technical reasons.

Reply:
Can we then maybe add a comment that explains this, so nobody will accidentally remove it in the future?

Reply:
added